src/main/jbake/content/documentation/bundles/models.md

```diff
@@ -85,6 +85,14 @@ Because the name of a constructor argument parameter cannot be detected via the
 
 Constructors may use any visibility modifier (as of [Sling Models 1.5.0](https://issues.apache.org/jira/browse/SLING-8069)).
 
+## @Model and adaptable types
+
+While technically it is possible to provide any class implementing the `Adaptable` interface as the `adaptable` parameter to the `@Model` annotation, the Sling Model Framework and the default injectors is mostly built under the assumption that the adaption is done from a Sling Resource. Some injectors also support adaption from a `SlingHttpServletRequest` object, but normally this is then converted into a adaption of the resource of the request.
```

Comment:
IMHO we should outline that all injectors can deal with adaptable SlingHttpServletRequest (potentially leveraging via) while Resource limits what injections are possible.

Reply:
Can you suggest an sentence? I feel a bit out of my comfort-zone here, especially when it comes to the ```@via``` annotation and it's replacement in the injector-specific annotations.

Reply:
@kwin: I want to give clear guidelines what to use; and recommending to always use SlingHttpServletRequest sounds incorrect, because then you cannot use it outside of a request, limiting the flexibility.

Reply:
The recommendation is fine but please mention that this comes with limitations. Some injectors just require a request…

Reply:
That should be already part of the description of the annotations.

Reply:
I rewrote this part here.